CHANGELOG.md

```diff
@@ -11,10 +11,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Changed
 - **Breaking (HTTP+COMIT API):** Change the identity for the Bitcoin Ledger from a public key hash to a public key. This change impacts the HTTP and the COMIT API of cnd.
 - **Breaking (COMIT API):**  Replace Ethereum `network` with Ethereum `chain_id`
-- cnd no longer automatically generates a config file but instead, simply defaults to what it would have written to the file on first startup.
+- cnd no longer automatically generates a config file, but instead simply defaults to what it would have written to the file on first startup.
```

Comment:
```suggestion
- cnd no longer automatically generates a config file, instead simply defaults to what it would have written to the file on first startup.
```